src/main/java/org/apache/bcel/classfile/Visitor.java

```diff
@@ -237,4 +249,15 @@ default void visitStackMapType(final StackMapType obj) {
     void visitSynthetic(Synthetic obj);
 
     void visitUnknown(Unknown obj);
+
+    /**
+     * Visits a {@link RecordComponentInfo} object.
+     *
+     * @param obj record component to visit
```

Comment:
This causes:
```
[ERROR] /Users/garydgregory/git/commons-bcel/src/main/java/org/apache/bcel/classfile/Visitor.java:256: error: @param name not found
[ERROR]      * @param obj record component to visit
[ERROR]               ^
[ERROR] 1 error
[ERROR] 100 warnings
```
when I run `mvn clean clean install site -P jacoco`

Reply:
Please rebase on git master to see this error in the CI build. For some reason, we had doclint disabled.

Reply:
@PabloNicolasDiaz 
You still have not fixed the above, the build fails.